tests/baselines/reference/outModuleConcatES6.sourcemap.txt

```diff
@@ -11,11 +11,89 @@ sourceFile:tests/cases/compiler/ref/a.ts
 >>>// This should be an error
 1 >
 2 >^^^^^^^^^^^^^^^^^^^^^^^^^^
-3 >                          ^^^^->
 1 >
   >
 2 >// This should be an error
 1 >Emitted(1, 1) Source(2, 1) + SourceIndex(0)
 2 >Emitted(1, 27) Source(2, 27) + SourceIndex(0)
 ---
+>>>export class A {

```

Comment:
This is another case where I'd rather we not accept this file as it uses `outFile`.